catalyst/dl/experiment/config.py

```diff
@@ -332,15 +331,17 @@ def _get_transform(**params) -> Callable:
                 for key, params_ in params.items()
             }
 
-            transform = AugmentorCompose({
-                key: Augmentor(
-                    dict_key=key,
-                    augment_fn=transform,
-                    input_key=key,
-                    output_key=key,
-                )
-                for key, transform in transforms_composition.items()
-            })
+            transform = AugmentorCompose(
+                {
+                    key: Augmentor(
+                        dict_key=key,
+                        augment_fn=transform,
+                        input_key=key,
+                        output_key=key,
+                    )
+                    for key, transform in transforms_composition.items()
+                }
+            )
```

Comment:
Is it possible to avoid the formatting of the code that your PR does not connect with?

Reply:
I removed this formating code, generated flake8, isoft. But now it doesn't pass codestyle-checks